src/OrleansPSUtils/project.json

```diff
@@ -0,0 +1,18 @@
+{
+  "dependencies": {
+    "Microsoft.CodeAnalysis.Analyzers": "1.0.0",
+    "Newtonsoft.Json": "7.0.1",
+    "Microsoft.CodeAnalysis.Common": "1.0.0",
+    "Microsoft.CodeAnalysis.CSharp": "1.0.0",
+    "System.Collections.Immutable": "1.1.36",
+    "System.Management.Automation.dll": "10.0.10586",
+    "System.Reflection.Metadata": "1.0.21"
+  },
```

Comment:
remember that we are now using transitive restore. That means that you need to reference only the packages that you care about, and not its dependencies. For example, you probably can just reference `Microsoft.CodeAnalysis.CSharp` and avoid `Microsoft.CodeAnalysis.Analyzers, Microsoft.CodeAnalysis.Common, System.Collections.Immutable and System.Reflection.Metadata`.
BTW, do you even require `Microsoft.CodeAnalysis.CSharp` or is this a mistake?